lib/login/login.dart

```diff
@@ -26,6 +27,7 @@ class _LoginPageState extends State<LoginPage> {
   String name;
   bool showLoading = false;
   UserAuthentication auth;
+  int repoNumber;
```

Comment:
we should rename this to something that is a bit more clear? maybe responseNumber?